lib/features/hooks/hooks.rb

```diff
@@ -54,25 +54,31 @@
     $logger.info LogUtil.linkify url, 'BrowserStack session(s)'
   end
   MazeRunner.driver.start_driver unless config.appium_session_isolation
+
+  # Call any blocks registered by the client
+  MazeRunner.hooks.call_after_configuration config
 end
 
 # Before each scenario
 Before do |scenario|
-  STDOUT.puts "--- Scenario: #{scenario.name}"
-  # TODO: PLAT-5309 - Building a clear environment for each scenario into MazeRunner would be a good
-  #   thing, but a number of our pipelines rely on certain variables existing throughout (e.g. Ruby).
-  # Runner.environment.clear
+  Runner.environment.clear
   Server.stored_requests.clear
   Store.values.clear
 
-  next if MazeRunner.config.farm == :none
+  unless MazeRunner.config.farm == :none
+    MazeRunner.driver.start_driver if MazeRunner.config.appium_session_isolation
+  end
```

Comment:
```suggestion
  if MazeRunner.config.farm != :none && MazeRunner.config.appium_session_isolation
    MazeRunner.driver.start_driver
  end
```

Reply:
Yep, and actually Rubocop then wants it to be a 1-liner.